build.gradle

```diff
@@ -6,46 +6,127 @@ import org.gradle.api.tasks.testing.logging.TestLogEvent
 
 plugins {
     id 'java'
+    id 'jacoco'
+    id 'checkstyle'
+    id 'com.github.kt3k.coveralls' version '2.4.0'
+    id 'com.github.johnrengelman.shadow' version '2.0.3'
     id 'org.asciidoctor.convert' version '1.5.6'
+    id 'application'
 }
 
+if (JavaVersion.current() == JavaVersion.VERSION_1_10
+        && System.getProperty('os.name').startsWith('Windows')) {
+    logger.warn('''\
+        ==============================================================================
+        *********************************** WARNING **********************************
+        ==============================================================================
+        You seem to be running Gradle with JDK 10 on Windows.
+        JDK 10 on Windows will fail to run tests in headless mode due to a JavaFX bug.
+        You are highly recommended to use JDK 9!
+        ==============================================================================
+        '''.stripIndent())
+}
+
+// Specifies the entry point of the application
+mainClassName = 'seedu.address.MainApp'
```

Comment:
Will build.gradle still be able to run since the entry point is no longer under seedu.address ?

Reply:
This is fixed in #14.